src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -24,144 +25,140 @@ function ProductGallery({ title, slug }: Props) {
   const { pages, state: searchState, addNextPage, addPrevPage } = useSearch()
   const { data } = useGalleryQuery()
 
-  const totalCount = data?.search.products.pageInfo.totalCount ?? 0
+  const totalCount = useTotalCount(data)
+
   const { next, prev } = usePagination(totalCount)
   const { isMobile } = useWindowDimensions()
 
   const [isFilterOpen, setIsFilterOpen] = useState<boolean>(false)
 
   const orderedFacets = useOrderedFacets(data)
 
-  if (!orderedFacets.length) {
-    return <div className="product-listing__data-loading">Loading…</div>
-  }
-
   return (
-    <div className="product-listing / grid-content-full">
-      <div className="product-listing__content-grid / grid-content">
-        <div className="product-listing__filters">
+    <>
+      <div className="product-listing__filters">
+        <Suspense fallback={null}>
```

Comment:
Just a question: Why did you change to use loadable at the Layout and to use Lazy here?

Reply:
I prefer to use the native `lazy`/`Suspense` whenever possible. 
But since we need to use `loadable` in some components of our `Layout` because if we didn't the build fails. There we switch to loadable (that deals better with Server Side).
What do you think?

Reply:
I think this is an nice thing to be documented, but not sure where 🙈  (this kind of decision)

Reply:
Maybe it's because I don't know the difference between `lazy` and `loadable`, but I also like the idea of documenting when to use each.
🙈

Reply:
[According to Gatsby](https://www.gatsbyjs.com/docs/how-to/performance/improving-site-performance/#step-5-on-critical-paths-lazy-load-below-the-fold-components):

> loadable-components is the recommended lazy-loading solution for all server-side-rendered React applications, including Gatsby websites.

So First I tried to use the native `lazy`/`Suspense`, but either the build failed or the application broke someplace. Thus, I switch to using the `loadable`, probably because there is some Server Side rendered dependency and it worked as aspected.

Finally, for that pages that use both, `lazy` and `loadable`, I kept the preference to use only `loadable` for the sake of avoiding loading 2 different things for the same purpose.

Where do you think is the best place to documented it?

Reply:
Since we seem to only have the README as the documentation, maybe it could be added under the **⚡ Performance & QA** section. At the same time, I'm not sure it's something we need to add there.
😕

Reply:
How about a comment in the code? (Like we have on `Sentinel` component) Or doesn't need it?

Reply:
I added this section on our `README.md` here 9bda2b62b32ae248059d6e0a7f39d9584ec72665.
Hmm, I think if we added the comment in the code we'll have a lot of comments, and maybe pollute the pages, for all places where we use lazy loading. 
What's are your opinions?

Reply:
Wonderful! 🌟